app/views/groups/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="create-group-nav">
+  <a href="<%= groups_path %>">
+    <i class="fa-solid fa-arrow-left"></i>
+  </a>
+  <h2>Create a new group</h2>
+</div>
+<div class="container-fluid w-100 d-flex flex-column align-items-center mt-5">
+  <%= form_with model: @group, url: groups_path, method: :post, :html => {:class => "w-75 mt-4"} do |form| %>
+  <div class="field-cont">
+    <div class="field">
+      <%= form.text_field :name, class:"input", placeholder:"Name", required: true %>
+    </div>
+    <div class="field">
+      <%= form.text_field :icon, class:"input", placeholder:"Icon link", required: true %>
+    </div>
+     <%= form.submit 'Create group', class: "create-btn create" %>
```

Comment:
- [ ] Great job validating your icon input field as the previous reviewer mentioned. However, there is one more thing that he mentioned that was left. I noticed that you didn't include error messages so users can be made aware of what might be causing the error they might be encountering instead of there being no error message at all :+1: 
- [ ] So kindly display the error messages when the input fields are left empty. :+1:

Reply:
Addressed